Makefile

```diff
@@ -0,0 +1,49 @@
+.PHONY: help fetch update cleanup dist publish
+SHELL=/bin/bash
+GEMSPEC=$(shell ls *.gemspec | head -1)
+VERSION=$(shell ruby -e 'puts Gem::Specification::load("$(GEMSPEC)").version')
+PROJECT=$(shell ruby -e 'puts Gem::Specification.load("$(GEMSPEC)").name')
+GEM=$(PROJECT)-$(VERSION).gem
+
+help: ## Print this message
+	@awk 'BEGIN { FS = ":.*##"; print "Usage:  make <target>\n\nTargets:" } \
+		/^[-_[:alpha:]]+:.?*##/ { printf "  %-15s%s\n", $$1, $$2 }' $(MAKEFILE_LIST)
+
+fetch: ## Grab latest from theme repo
```

Comment:
Rather than calling `make cleanup` in this target, have it depend on the `cleanup` target:
```
fetch: cleanup ## Grab latest from theme repo
```

Reply:
I was trying to do that and failed. This is exactly the type of feedback I was looking for. Thanks!